telnet.js

```diff
@@ -0,0 +1,193 @@
+const blessed     = require('blessed');
+const telnet      = require('telnet2');
+const axios       = require('axios');
+const covid19     = require('./lib/cli');
+const pkg         = require('./package.json');    // package.json info
+const apiBaseURL  = "https://corona.lmao.ninja";  // NovelCOVID API
+const port        = process.env.TELNET_PORT || 2300;     // set port
+const interval    = process.env.INTERVAL || 2000
+
+async function fetchGlobalTracker() {
+  const { data } = await axios.get(`${apiBaseURL}/all`);
+        
+  return covid19.covid19globaltracker(
+    data.cases, data.deaths,
+    data.recovered, data.updated
+  );
+}
+
+async function fetchByCountry(query) {
+  // empty, country, history, chartType
+  const [_x, country, history, chartType] = query.split('/'),
+        countryData = await axios.get(`${apiBaseURL}/countries/${country}`),
+        all = await axios.get(`${apiBaseURL}/all`),
```

Comment:
We can omit this call now since we only call this to get the "updated" attribute. 
The country API has `updated` attribute available now.

so instead of `u.updated` use `d.updated` on line 37 same with line 45

Reply:
Hi I already removed the extra API call on country tracker. Can you please verify if this is correct. Thanks!

Reply:
@ianvizarra please verify this and close if necessary so we can move forward. Will improve this later on.

Reply:
outdated nato...